.gitignore

```diff
@@ -1,3 +1,5 @@
 *~
 playbooks/group_vars/all
 playbooks/db-change-record-set.json
+playbooks/roles/creategroups/files/s3-policy.json
+playbooks/s3-policy.json
```

Comment:
Désolé Seb, j'ai accidentellement déplacé mon répertoire /etc/* dans dbs.internal donc mon PC était mort il a fallu que je réinstalle tout. Là j'essaie de remettre ma configuration ancienne et installer ansible et tout pour continuer
 ---------------------------------------------------
CAMARA Alpha Lalmas
DevOps Freelance
Lalmas Group's founder(Guinea)
Allô Taxi's founder(Guinea)
twitter: https://twitter.com/lalmas_camara
fr.linkedin.com/in/lalmascamara/en
skype: alpha-lalmas.camara 


     Le Mardi 3 novembre 2015 1h49, Sebastien Mirolo <notifications@github.com> a écrit :
   

 In .gitignore:> @@ -1,3 +1,5 @@
>  *~
>  playbooks/group_vars/all
>  playbooks/db-change-record-set.json
> +playbooks/roles/creategroups/files/s3-policy.json
> +playbooks/s3-policy.json
Why are those 2 files in .gitignore? I would guess one of them could be but both?—
Reply to this email directly or view it on GitHub.